hack/lib/catalogsource.bash

```diff
@@ -71,6 +71,12 @@ function install_catalogsource {
     mv "${rootdir}/_output/bkp.Dockerfile" "${index_build_dir}/Dockerfile"
   fi
 
+# IMAGE_INDEX_BUNDLE decleare in var.bash as default null then work like existing if not null it will take the value from script iib images
+if [ -n "${IMAGE_INDEX_BUNDLE:-}" ]; then
+    echo "Image index is : $IMAGE_INDEX_BUNDLE"
+    index_image="$IMAGE_INDEX_BUNDLE"
+fi
```

Comment:
Can you replaces the above with this?
```
if [ -n "${INDEX_IMAGE:-}" ]; then
    echo "Index image : $INDEX_IMAGE"
    index_image="$INDEX_IMAGE"
fi
```
and also change the name in vars.bash
Notes: We already use "serverless-index" and "serverless-bundle", these are separate images. The name you have (INDEX_IMAGE_BUNDLE) looks like it combines them together. It's a bit confusing.
The comment starting with "IMAGE_INDEX_BUNDLE decleare" is probably unnecessary, it's really clear from the code what it does.
Nit: The indentation was wrong, it needs to be shifted right.